sdv/constraints/tabular.py

```diff
@@ -389,6 +387,17 @@ def reverse_transform(self, table_data):
         """
         table_data = table_data.copy()
         diff_column = np.exp(table_data[self._diff_column_name]) - 1
+
+        # Clip the column to ensure the reverse transformed values satisfy the constraint
+        if self._operator is np.greater_equal:
+            diff_column = diff_column.clip(0)
+        elif self._dtype == np.dtype('int'):
+            diff_column = diff_column.clip(1)
+        elif self._dtype == np.dtype('float'):
+            diff_column = diff_column.clip(1e-6)
+        elif self._is_datetime:
+            diff_column = diff_column.clip(1_000_000_000)
```

Comment:
Datetimes have the same problem as the floats: what's the minimal unit we want to increase it by? Here I'm increasing by 1sec, but maybe the user wants to increase by 1 day? 1 year? 1 nanosecond? 

Again, I don't think this matters too much, as this very rarely (if ever) happens, so I just clip to 1sec.